extension/src/test/suite/plots/index.test.ts

```diff
@@ -763,14 +764,18 @@ suite('Plots Test Suite', () => {
     }).timeout(WEBVIEW_TEST_TIMEOUT)
 
     it('should send the correct data to the webview for flexible plots', async () => {
-      const { plots, messageSpy, mockPlotsDiff, experiments } =
-        await buildPlots(disposable, multiSourcePlotsDiffFixture)
-
-      stub(experiments, 'getSelectedRevisions').returns([
+      stub(Experiments.prototype, 'getSelectedRevisions').returns([
         { label: 'workspace' },
         { label: 'main' }
       ] as SelectedExperimentWithColor[])
 
+      stub(Experiments.prototype, 'getBranchRevisions').returns([])
```

Comment:
[F] Multi-source test fixture doesn't match the default exp show fixture. The change required would blow this PR out so I have just patched it for now. Will follow-up once this is merged.

Reply:
Thank you!